core/src/main/scala/org/apache/spark/shuffle/ShuffleManager.scala

```diff
@@ -60,6 +60,9 @@ private[spark] trait ShuffleManager {
    */
   def shuffleBlockResolver: ShuffleBlockResolver
 
+  /** Return short name for the ShuffleManager */
+  def shortShuffleMgrName: String
```

Comment:
I would make these `val`s since we don't ever expect them to change within a shuffle manager